recipes/aws-c-cal/all/conanfile.py

```diff
@@ -83,12 +81,13 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "aws-c-cal"))
 
     def package_info(self):
-        self.cpp_info.filenames["cmake_find_package"] = "aws-c-cal"
-        self.cpp_info.filenames["cmake_find_package_multi"] = "aws-c-cal"
+        self.cpp_info.set_property("cmake_file_name", "aws-c-cal")
         self.cpp_info.names["cmake_find_package"] = "AWS"
         self.cpp_info.names["cmake_find_package_multi"] = "AWS"
+        self.cpp_info.set_property("cmake_target_name", "AWS")
         self.cpp_info.components["aws-c-cal-lib"].names["cmake_find_package"] = "aws-c-cal"
         self.cpp_info.components["aws-c-cal-lib"].names["cmake_find_package_multi"] = "aws-c-cal"
+        self.cpp_info.components["aws-c-cal-lib"].set_property("cmake_target_name", "aws-c-cal")
```

Comment:
```suggestion
        self.cpp_info.components["aws-c-cal-lib"].set_property("cmake_target_name", "AWS::aws-c-cal")
```

Reply:
same as on the other PR. this requires conan 1.43.0 which is not yet deployed on CCI, so feel free to ignore these suggestions